ts/data.ts

```diff
@@ -1,6 +1,22 @@
 const data = {
   view: 'entry-form',
-  entries: [],
+  entries: readData(),
   editing: null,
   nextEntryId: 1,
 };
```

Comment:
You should also create an `interface` for `Data`. It will fix probably some of the type errors you ran into